From: Maximiliano Curia <maxy@gnuservers.com.ar>
Date: Sat, 2 Sep 2017 15:12:29 +0200
Subject: Use a scoped builtin_version file

---
 stgit/version.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/stgit/version.py b/stgit/version.py
index 7cac6f6..c52fc61 100644
--- a/stgit/version.py
+++ b/stgit/version.py
@@ -25,7 +25,7 @@ def git_describe_version():
 
 def builtin_version():
     try:
-        import builtin_version as bv
+        import stgit.builtin_version as bv
     except ImportError:
         raise VersionUnavailable()
     else:
